raiden/network/rpc/client.py

```diff
@@ -361,6 +366,9 @@ def manager_by_token(self, token_address):
         if token_address not in self.token_to_channelmanager:
             token = self.token(token_address)  # check that the token exists
             manager_address = self.default_registry.manager_address_by_token(token.address)
+            if manager_address == '':
```

Comment:
By making this change you also need to then change [this](https://github.com/raiden-network/raiden/blob/master/raiden/api/python.py#L84) line to catch the `NoTokenManager` error. You actually get a failing unit test because of that.